Makefile

```diff
@@ -71,6 +71,12 @@ fly-windows-units:
 format: ## Run go fmt
 	go fmt ./...
 
+install-test-deps: ## Install "global" dependencies needed to run tests
+# Running `go install <pkg>` without a version specifier will use version specified in go.mod
+# See https://go.dev/ref/mod#go-install
+	go install github.com/onsi/ginkgo/v2/ginkgo
+	go install github.com/onsi/gomega/matchers
```

Comment:
**question/non-blocking** Unrelated directly to this change, but why do we install `github.com/onsi/gomega/matchers`? I don't believe thats a binary, its just the library used in the test cases for assertions.